DittoToolsApp/DittoToolsApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -9,6 +9,15 @@
         "version" : "3.0.4"
       }
     },
+    {
```

Comment:
I don't think we want this because @phatblat had originally removed this so a developer working on the tools could reference the local version of the package rather than fix themselves to a mainline revision. Is that the idea ben?

Reply:
Correct. This makes the app use the `main` branch of the repo instead of the current one. The only way to test changes before merging them is to have the app point to the local package via a filesystem path.

It's hard to read the diffs on this project file, but the `path = ..` above on [line 93](https://github.com/getditto/DittoSwiftTools/pull/32/files#diff-54bca7190278135094a57bc21b7a0761003dd65154f0fad4c0be10e9416adaa6L93) is what makes the project use the local package.

```
F87DC47A2988581000899FEC /* DittoSwiftTools */ = {isa = PBXFileReference; lastKnownFileType = wrapper; name = DittoSwiftTools; path = ..; sourceTree = "<group>"; };
```